test/local_test.dart

```diff
@@ -29,89 +30,48 @@ void main() {
       tmp.deleteSync(recursive: true);
     });
 
+    setUpAll(() {
+      if (io.Platform.isWindows) {
+        ignoreOsErrorCodes = true;
```

Comment:
Should we file an issue and add a TODO here to investigate/remove this? Not a high-prio item, but might be worth tracking.